quick-start/api/swagger/mocks.json

```diff
@@ -130,6 +130,207 @@
           }
         }
       }
+    },
+    "/mapping-process": {
```

Comment:
Should the URL be `/mapping-process`? I think the scheme should be `/process` and fileters for type should be params, like `/process?type=mapping`